packages/fuselage/src/components/Box/Scrollable/index.js

```diff
@@ -1,9 +1,24 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useCallback } from 'react';
 
 import { useProps } from '../../../hooks';
 
-export function Scrollable({ children, horizontal, vertical, smooth }) {
+export function Scrollable({ children, horizontal, vertical, smooth, scrollCb }) {
+  let scrollTimeout;
```

Comment:
Prefer to share variables across re-renders via `useRef`: declare with `const scrollTimeoutRef = useRef();` and use it as `scrollTimeoutRef.current`. Scopes of functional components should be as discardable as possible to avoid memory leaks; keep only on the hooked state.